vendor/wp-now/src/wp-now.ts

```diff
@@ -617,6 +617,23 @@ async function mountInternalMuPlugins( php: PHP, options: WPNowOptions ) {
 			`
 		);
 	}
+
+	php.writeFile(
+		path.posix.join( PLAYGROUND_INTERNAL_MU_PLUGINS_FOLDER, '0-http-request-timeout.php' ),
+		`<?php
+		// Set the default HTTP request timeout to 60 seconds
+		add_filter( 'http_request_timeout', function() {
+			return 60;
+		} );
+
+		// Set the default cURL timeout to 60 second
```

Comment:
The first filter looks self-explanatory and the second one is pretty self-explanatory too.
So I would propose to remove *both* comments and rather provide one comment at the beginning with info about the *reason* for increasing timeouts.

```suggestion
		// Updating plugins locally fails with defaulk timeout and 60 secs looks enough

		add_filter( 'http_request_timeout', function() {
			return 60;
		} );

```

Reply:
That's a good point @nightnei, thanks for highlighting. Done in [771b954](https://github.com/Automattic/studio/pull/1407/commits/771b9548c3c3dc8298a6970508cdbcd3392f1c6a).